test/ILLink.RoslynAnalyzer.Tests/ComAnalyzerTests.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using ILLink.Shared;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+using VerifyCS = ILLink.RoslynAnalyzer.Tests.CSharpAnalyzerVerifier<
+	ILLink.RoslynAnalyzer.COMAnalyzer>;
+
+namespace ILLink.RoslynAnalyzer.Tests
+{
+	public class ComAnalyzerTests
+	{
+		static Task VerifyComAnalyzer (string source, params DiagnosticResult[] expected)
+		{
+			return VerifyCS.VerifyAnalyzerAsync (source,
+				TestCaseUtils.UseMSBuildProperties (MSBuildPropertyOptionNames.EnableTrimAnalyzer),
+				expected: expected);
+		}
+
+		[Fact]
+		public Task Issue ()
+		{
+			var TargetParameterWithAnnotations = @"
+using System.Runtime.InteropServices;
+
+static class Program
+{
+    [DllImport(""Unimportant"")]
+    static extern unsafe void WhateverName(void* pointerArg);
+
+    static void Main()
+    {
+        unsafe
+        {
+            WhateverName(null);
+        }
+    }
+}";
+			return VerifyComAnalyzer (TargetParameterWithAnnotations,
```

Comment:
Im wondering if it would be better to have it as a shared test with linker. Maybe inside [test](https://github.com/dotnet/linker/blob/a372828948529c86f3ec02db9ddd9ce7ce1d8b06/test/Mono.Linker.Tests.Cases/Interop/PInvoke/Warnings/ComPInvokeWarning.cs)?

Reply:
Yes - pleas add this a shared test.